filter_plugins/oo_filters.py

```diff
@@ -621,11 +621,23 @@ def to_padded_yaml(data, level=0, indent=2, **kw):
             return ""
 
         try:
-            transformed = yaml.safe_dump(data, indent=indent, allow_unicode=True, default_flow_style=False, **kw)
+            if isinstance(data, list):
+                _data = []
+                for d in data:
+                    _data.append(unicode(d))
+            elif isinstance(data, dict):
+                # Assumes all values are simple strings
+                _data = {}
+                for k, v in data.iteritems():
+                    _data[unicode(k)] = unicode(v)
+            else:
+                _data = data
+
```

Comment:
After sleeping on this, I wonder if it makes sense to generalize this logic into some kind of utility function? It might end up being reusable in the future in other template situations.